content/docs/observability/deployment/offline.md

```diff
@@ -106,15 +106,15 @@ To perform an offline installation of a Helm chart, the following steps should b
     ```bash
     ./offline-installer.sh -p <my-registry>:5000
     ```
-    ```  
+    ```
     *
     * Loading, tagging, and pushing Docker images to registry <my-registry>:5000/
 
-      dellemc/csm-topology:v1.5.0 -> <my-registry>:5000/csm-topology:v1.5.0
-      dellemc/csm-metrics-powerflex:v1.5.0 -> <my-registry>:5000/csm-metrics-powerflex:v1.5.0
-      dellemc/csm-metrics-powerstore:v1.5.0 -> <my-registry>:5000/csm-metrics-powerstore:v1.5.0
-      dellemc/csm-metrics-powerscale:v1.2.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.2.0
-      dellemc/csm-metrics-powermax:v1.0.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.0.0
+      dellemc/csm-topology:v1.6.0 -> <my-registry>:5000/csm-topology:v1.6.0
+      dellemc/csm-metrics-powerflex:v1.6.0 -> <my-registry>:5000/csm-metrics-powerflex:v1.6.0
+      dellemc/csm-metrics-powerstore:v1.6.0 -> <my-registry>:5000/csm-metrics-powerstore:v1.6.0
+      dellemc/csm-metrics-powerscale:v1.3.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.3.0
+      dellemc/csm-metrics-powermax:v1.1.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.1.0
```

Comment:
On the right side, change powerscale to powermax.

Reply:
Fixed.